### PR TITLE
1 check oss issue tracker

### PR DESCRIPTION
**Pull request by gbataille**
Obviously not finished but:
- I'd like to get comments
- We might still merge it to have the basic skeleton for checkers and for test organization

### CODE REVIEW

File: tests/Test/Krank/Checkers/IssueTrackerSpec.hs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import Text.Regex.Applicative ((=~))
+
+import Krank.Checkers.IssueTracker
+
+spec :: Spec
+spec = do
+  context "Test.Krank.Checkers.specIssueTracker" $ do
+    describe "#githubRE" $ do
+      it "handles full https url" $ do
+        "https://github.com/guibou/krank/issues/1" =~ githubRE `shouldBe` Just 1
```
Comment (guibou) [inline tests/Test/Krank/Checkers/IssueTrackerSpec.hs:16]: Cool. I'd see something such as

```haskell
$ do
   let matchGithubRe url = url =~ githubRE `shouldBe` Just 1
```

But that's fine as it is too.

File: src/Krank/Checkers/IssueTracker.hs [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{-# LANGUAGE ApplicativeDo #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+module Krank.Checkers.IssueTracker (
```
Comment (guibou) [inline src/Krank/Checkers/IssueTracker.hs:4]: module `GithubIssueTracker` perhaps? Or do you think we'll extend this to other tool (such as gitlab) soon?

**Review by guibou — APPROVED** [review]

File: tests/Test/Krank/Checkers/IssueTrackerSpec.hs (hunk 1 above)
Comment (gbataille) [reply tests/Test/Krank/Checkers/IssueTrackerSpec.hs:16]: why? I'm not seeing the point of capturing that. Actually would that capture the spec or the match?

File: src/Krank/Checkers/IssueTracker.hs (hunk 2 above)
Comment (gbataille) [reply src/Krank/Checkers/IssueTracker.hs:4]: My goal was to put a Gitlab RE in the same file indeed

Comment (guibou) [reply src/Krank/Checkers/IssueTracker.hs:4]: Good !

**Comment by guibou** [comment]
Go merge!

**Pull request merged**